cluster/charts/rook-cluster/templates/NOTES.txt

```diff
@@ -0,0 +1 @@
+The Rook Cluster has been installed.
```

Comment:
Can we add some more detail here about how to use the cluster after installation?